packages.dhall

```diff
@@ -1,23 +1,12 @@
 let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.13.6-20200123/packages.dhall sha256:687bb9a2d38f2026a89772c47390d02939340b01e31aaa22de9247eadd64af05
-
-let overrides =
-      { halogen = upstream.halogen // { version = "v5.0.0-rc.7" }
-      , halogen-vdom = upstream.halogen-vdom // { version = "v6.1.0" }
-      }
+      https://github.com/purescript/package-sets/releases/download/psc-0.13.6-20200507/packages.dhall sha256:9c1e8951e721b79de1de551f31ecb5a339e82bbd43300eb5ccfb1bf8cf7bbd62
 
 let additions =
-  { subcategory =
-      { dependencies =
-          [ "prelude"
-          , "profunctor"
-          , "record"
-          ]
-      , repo =
-          "https://github.com/matthew-hilty/purescript-subcategory.git"
-      , version =
-          "v0.2.0"
+      { subcategory =
+        { dependencies = [ "prelude", "profunctor", "record" ]
+        , repo = "https://github.com/matthew-hilty/purescript-subcategory.git"
+        , version = "v0.2.0"
+        }
```

Comment:
I didn't manually change this formatting. I believe spago did that when I used it to upgrade the package set.

Reply:
👍 that's fine